src/blocks/video.js

```diff
@@ -72,6 +74,13 @@ module.exports = Block.extend({
   onDrop: function(transferData){
     var url = transferData.getData('text/plain');
     this.handleDropPaste(url);
+  },
+
+  asClipboardHTML: function() {
+    var data = this.getBlockData();
+    var source = this.providers[data.source];
+    var src = source.url(data.remote_id);
+    return `<p>${src}</p>`;
```

Comment:
Same here - couldn't the HTML be the iframe?

Reply:
As in, fetching the source code of the URL loaded in the `iframe`? I actually like this approach, as it sort of matches how I write articles even outside of Sir Trevor.

Reply:
@ilyaporopudas this comes from something i was talking to @callum about regarding the different reasons for copy / paste. At the moment i've built it for export to another system, but we'll want to include a text representation and also an internal version for when pasting back into sir-trevor.

Reply:
👍